Simple-Projects/Simple_Projects/Pen&Paper/Paper.cpp

```diff
@@ -1,37 +1,37 @@
+#include <iostream>
 #include "Paper.h"
 
-Paper::Paper(int maxSymbols): maxSymbols(4096) {
-    this->maxSymbols = maxSymbols;
-    this->symbols = 0;
-    this->content = "";
+Paper::Paper(size_t maxSymbols): maxSymbols(maxSymbols), symbols(0) {
 }
 
-Paper::~Paper() {}
+Paper::~Paper() = default;
 
-int Paper::getMaxSymbols() const {
+size_t Paper::getMaxSymbols() const {
```

Comment:
Since you're working with a string's size here you'll have to match its size type. A signed int just won't cut it